### ISSUE TITLE
ID 261597: [Screen reader - Simon – Alert rules]: Heading hierarchy is not sequential in "Alert rules" tab present under "Alerting" screen.

### ISSUE THREAD

**Issue by natydej** [grafana/grafana#67040]
### **User Experience:** 
Visual Impaired users will be unable to understand and face difficulty in relating the content while navigating the page in heading mode of the screen reader as the heading structure/order is not logical. 
_Note: User credentials should NOT be included in the bug._

**Pre-requisite:**
Turn on Narrator.

### **Repro steps:**

1. Open [https://play.grafana.org/](https://play.grafana.org/) Sign in with valid credentials. "Home" page appears. 
2. Navigate to the left navigation bar and to the "Alerting" button, select it.
3. It gets expanded and navigates to the expanded sub menu items.
4. Select the "Alert Rules" menu items. A new page "Alerting" appears.
5. Navigate the tab " Alert Rules". Verify the issue

**Actual Result:** 
 Issue:  Heading hierarchy is not defined sequentially in **"Alert rules"** tab present under **"Alerting"** screen.

**Observation**: 
 "Grafana" heading and the Contents(headings) such as "Data source: AWS IoT Site-Wise" present under it, is defined under `<H5>` and `<H6>` heading level. Which is defined just after `<H1>` heading level.

**Expected Result:** 
Heading hierarchy should be defined sequentially in in "Alert rules" tab present under "Alerting" screen. "Grafana" heading and the Contents(headings) such as "Data source: AWS IoT Site-Wise" present under it, should be defined under `<H2>` and `<H3>` heading level.

_Note: The same issue exists with the "Mimir/Cortex/Loki" heading section under the same "Alert rules" tab present under the "Alerting" screen._

**Screenshot:**
<img width="548" alt="Screenshot 2023-04-21 at 12 25 27" src="https://user-images.githubusercontent.com/51436795/233615022-f2947d06-deb5-4508-a9a7-f2dc96747ff7.png">

### **Severity:**
`SEV2`

### **Area:**
`Alert Rules` (GrafanaRules Component)

**Comment by khushijain21** [comment]
Hello, I'd like to work on this, can you guide to the file location?
